shakenfist/client/main.py

```diff
@@ -704,7 +705,12 @@ def instance_create(ctx, name=None, cpus=None, memory=None, network=None, networ
 
     diskdefs = []
     for d in disk:
-        size, base = _parse_spec(d)
+        p = _parse_spec(d)
+        if not p[1]:
+            print('Disk short-hand specification is <size>@<base>')
+            return
```

Comment:
I don't think this is true. We should allow short hand blank disks. For example, ephemeral data disks. This is a valid start command:

```
sf-client instance create foo 1 1024 -d 8@cirros -d 42 -n ...
```

Reply:
True. Good save.